en: add helm offline doc (#503)
pingcap/docs-tidb-operator: en/tidb-toolkit.md

    helm init --upgrade
    ```

3. Configure the Helm Repository

    Kubernetes applications are packed as chart in Helm. PingCAP provides the following Helm charts for TiDB in Kubernetes:

    * `tidb-operator`: used to deploy TiDB Operator;
    * `tidb-cluster`: used to deploy TiDB clusters;

>     * `tidb-backup`: used to backup or restore TiDB clusters;

Author: OK, thanks!

    * `tidb-lightning`: used to import data into a TiDB cluster;
    * `tidb-drainer`: used to deploy TiDB Drainer;
    * `tikv-importer`: used to deploy TiKV Importer.

>     These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local using the following command:

Author: ```suggestion
    These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local server or computer using the following command:
```

>     These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local using the following command:

Author: updated

    {{< copyable "shell-regular" >}}

    ```shell
    helm repo add pingcap https://charts.pingcap.org/
    ```

    After adding, use `helm search` to search for the charts provided by PingCAP:

    - If the Helm version < 2.16.0:

        {{< copyable "shell-regular" >}}

        ```shell
        helm search pingcap -l
        ```

    - If the Helm version >= 2.16.0:

        {{< copyable "shell-regular" >}}

        ```shell
        helm search pingcap -l --devel
        ```

    ```
    NAME                    CHART VERSION   APP VERSION DESCRIPTION
    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
    ...

>     ```

Author: The output is for the above two cases, so I think no need to indent more here.

    When a new version of chart has been released, you can use `helm repo update` to update the repository cached locally:

    {{< copyable "shell-regular" >}}

    ```shell
    helm repo update
    ```

>     Common Helm operations include `helm install`, `helm upgrade`, and `helm del`. Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.

Reviewer: ```suggestion
    Common Helm operations include `helm install`, `helm upgrade`, `helm del`, and `helm ls`. Helm charts usually contain many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure these parameters using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
```

> 

Reviewer: ```
    执行部署、升级、销毁等操作前，可以使用 `helm ls` 查看集群中已部署的应用：

    {{< copyable "shell-regular" >}}

    ```shell
    helm ls
    ```
```

These lines are missing?

Author: updated

>     When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can use `chart-version` to specify the chart version (by default the latest GA is used). The steps in command line are as follows:

Reviewer: ```suggestion
    When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can specify `chart-version` by using the `--version` flag to choose a specific chart version (by default the latest GA is used). The steps in command line are as follows:
```

    * Install:

        {{< copyable "shell-regular" >}}

        ```shell
        helm install ${chart_name} --name=${release_name} --namespace=${namespace} --version=${chart_version} -f ${values_file}
        ```

>     * Upgrade (upgrade can be done by modifying the `chart-version` to upgrade to the latest chart version or the `values.yaml` file to update the configuration):

Reviewer: ```suggestion
    * Upgrade (the upgrade can be either modifying the `chart-version` to upgrade to the latest chart version, or editing the `values.yaml` file to update the configuration):
```

        {{< copyable "shell-regular" >}}

        ```shell
        helm upgrade ${release_name} ${chart_name} --version=${chart_version} -f ${values_file}
        ```

    * Delete:
    
        To delete the application deployed by Helm, run the following command:

        {{< copyable "shell-regular" >}}

        ```shell
        helm del --purge ${release_name}
        ```

    For more information on Helm, refer to [Helm Documentation](https://helm.sh/docs/).

4. Use Helm chart offline

>     If the server cannot access the Internet, you cannot install the TiDB Operator or other applications through configuring the Helm repository. In this case, you have to download the Helm charts required during the cluster installation on a server or computer that has Internet access, and then upload them to the server.

Reviewer: ```suggestion
    If the server cannot access the Internet, you cannot install TiDB Operator or other applications through configuring the Helm repository. In this case, you have to download the Helm charts required during the cluster installation on a server or computer that has Internet access, and then upload them to the server.
```

        helm search pingcap -l
        ```

    - If the Helm version >= 2.16.0:

        {{< copyable "shell-regular" >}}

        ```shell
        helm search pingcap -l --devel
        ```

    ```
    NAME                    CHART VERSION   APP VERSION DESCRIPTION
    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
    ...
    ```

    When a new version of chart has been released, you can use `helm repo update` to update the repository cached locally:

    {{< copyable "shell-regular" >}}

    ```shell
    helm repo update
    ```

    Common Helm operations include `helm install`, `helm upgrade`, `helm del`, and `helm ls`. The Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure these parameters using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.

>     Before the operations of installation, upgrade, or deletion, etc., you can execute `helm ls` to view the applications that have been installed in the cluster:

Reviewer: ```suggestion

    Before the operations of installation, upgrade, deletion, and so on, you can execute `helm ls` to view the applications that have been installed in the cluster:
```
Avoid the use of Latin words such as `etc`, `i.e.`, and `e.g.`.

Author: OK, thanks!
